Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -83,7 +87,7 @@ public class ConsensusLoop
         private readonly Signals.Signals signals;
 
         /// <summary>A lock object that synchronizes access to the <see cref="ConsensusLoop.AcceptBlock"/> and the reorg part of <see cref="ConsensusLoop.PullerLoop"/> methods.</summary>
-        private readonly object consensusLock;
+        private readonly AsyncLock consensusLock;
```

Comment:
Why?

Reply:
half way to the next issue i guess
but personal dislike of that old lock there :)

Reply:
ok as you are going to make this async soon, then I agree. but I prefer the lock because AsyncLock does not bring any benefit.